reducers/workoutRecordReducer.js

```diff
@@ -0,0 +1,21 @@
+import { ADD_WORKOUT_RECORD } from '../constants/index';
+
+const initialState = {
+   records: [],
+}
+
+const recordReducer = (state = initialState, action) => {
+   switch(action.type) {
+        case ADD_WORKOUT_RECORD:
+            console.log(`Adding workout to store with ${action.record}`);
+            const newRecords = [...state.records];
+            newRecords.push(action.record)
+            return {
+                records: newRecords,
+            };
```

Comment:
Did you guys consider storing the records in an object like a hashmap with the dates as the key and the record as the value. Cause we're not gonna show a list of the records anywhere right so this will make it faster to retrieve

Reply:
This isn't a huge deal though and we can just fix this going forward

Reply:
yeah I was thinking about that didn't get around to it yet